modules/core/src/main/java/org/apache/synapse/mediators/v2/Utils.java

```diff
@@ -0,0 +1,50 @@
+/*
+ *  Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com) All Rights Reserved.
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied. See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.apache.synapse.mediators.v2;
+
+import org.apache.synapse.MessageContext;
+import org.apache.synapse.SynapseConstants;
+
+public class Utils {
```

Comment:
shall we rename to a name like foreachUtils since we are not inside a dedicated package and this util is not common to all v2 mediators

Reply:
Updated with 477dfd6